app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Eav.php

```diff
@@ -260,7 +268,8 @@ public function __construct(
         $attributesToEliminate = [],
         CompositeConfigProcessor $wysiwygConfigProcessor = null,
         ScopeConfigInterface $scopeConfig = null,
-        AttributeCollectionFactory $attributeCollectionFactory = null
+        AttributeCollectionFactory $attributeCollectionFactory = null,
+        Url $urlBuilder
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@sidolov 
I have just resolved this. Thanks for the feedback